src/Cake.Npm/NpmPruneAliases.cs

```diff
@@ -0,0 +1,146 @@
+        public static void NpmPrune(this ICakeContext context, IEnumerable<string> arguments)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            if (arguments == null)
+            {
+                throw new ArgumentNullException(nameof(arguments));
+            }
+
+            var settings = new NpmPruneSettings();
+
+            foreach (var argument in arguments)
+            {
+                settings.Arguments.Add(argument);
+            }
+
+            context.NpmPrune(settings);
+        }
+
+        /// <summary>
+        /// Runs npm prune using the settings returned by a configurator.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="configurator">The settings configurator.</param>
+        /// <example>
+        /// <para>Use specific log level</para>
+        /// <code>
+        /// <![CDATA[
+        ///     NpmPrune(settings => settings.WithLogLevel(NpmLogLevel.Verbose));
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Prune")]
+        public static void NpmPrune(this ICakeContext context, Action<NpmPruneSettings> configurator)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            if (configurator == null)
+            {
+                throw new ArgumentNullException(nameof(configurator));
+            }
+
+            var settings = new NpmPruneSettings();
+            configurator(settings);
+            context.NpmPrune(settings);
+        }
+
+        /// <summary>
+        /// Runs npm prune with the specified settings.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <param name="settings">The settings.</param>
+        /// <example>
+        /// <para>Use specific log level</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var settings = 
+        ///         new NpmPruneSettings 
+        ///         {
+        ///             LogLevel = NpmLogLevel.Verbose
+        ///         };
+        ///     NpmPrune(settings);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Prune")]
+        public static void NpmPrune(this ICakeContext context, NpmPruneSettings settings)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            AddinInformation.LogVersionInformation(context.Log);
+
+            var packer = new NpmPruneRunner(context.FileSystem, context.Environment, context.ProcessRunner, context.Tools, context.Log);
```

Comment:
Variable should be named appropriately

Reply:
This has been renamed, as well as some file names I missed on the first run-through